apps/coscrad-frontend/term-list.container.tsx

```diff
@@ -85,7 +81,7 @@ export const TermListContainer = (): JSX.Element => {
         <TermIndexTable
             type={AggregateType.term}
             headingLabels={headingLabels}
-            tableData={terms as ITermViewModel[]}
+            tableData={(terms as ITermViewModel[]) || []}
```

Comment:
I wonder why this was necessary. Did we miss a loading \ error check above?

Reply:
I think I sorted that out. I need to run Cypress again to be sure my solution works.

Reply:
Cypress is still green.